.github/workflows/mac-tests.yml

```
@@ -121,7 +121,7 @@ jobs:
 
     - name: Tests
       run: |
-        pytest tests -m "integration"
+        pytest tests -m "integration" --ignore-glob="allennlp"
```

How about following the way to ignore tests in `tests-integration.yml`?

https://github.com/optuna/optuna/blob/01aa6d9733269aa58e0ae8931a1bd28b662d7942/.github/workflows/tests-integration.yml#LL121C10-L121C60

```suggestion
        pytest tests -m "integration" --ignore tests/integration_tests/allennlp_tests
```

---

Thank you. That looks better and I fix it.

---

Thanks!